### DISCUSSION TITLE
MeshCentral 1.1.33 Released, BETA bootstrap interface!

### DISCUSSION THREAD

**Discussion by si458** [Ylianst/MeshCentral#6493] [General]
MeshCentral 1.1.33 has been released and it includes a preview of a new bootstrap interface!
you can set `siteStyle:3` under domains in your `config.json` OR add `?sitestyle=3` in your url bar!
We hope everyone can try the new version of the interface and give us feedback!
The interface will not replace the old one but run along side it!

**Comment by timcanty** [comment]
Curiosity got the better of me, and had to give this a go. Looks good as the first public beta. Thank you all for the effort gone into this.

Do you want feedback in here or opening a issue?

Such as if you click to edit a device name, it brings up a dialogue about editing tags instead.
![image](https://github.com/user-attachments/assets/b6872704-93c2-45ab-8be8-599f73a85f35)

and if you go full screen the size of the bar isn;t correct to the size of the buttons
![image](https://github.com/user-attachments/assets/453f0311-ecba-43fd-80ad-dedcd2fc5784)

They was just a few early ones I could see.

**Reply by si458** [reply]
no thats fine leave them here! the will indeed be a few bugs!
im aware of those 2 exact bugs and im sure i fixed the first one,
might of forgotten to commit the fixes! will have alook during the week 👍 
any others your find just leave tag into your post here 👍

**Comment by PathfinderNetworks** [comment]
I just upgraded and enabled this on one of my MeshCentral servers.  It is working but I'm noticing some odd quirks so far.  For starters, is there any way or option to decrease the overall size of everything (fonts, icons, etc)?  I don't know the exact percentage, but it appears the bootstrap version has most items (although not all) a good 10-15% larger than the non-bootstrap version.  This is making it so that the screen space is not optimized, especially when managing a large number of devices

When connected to a device and you go to full screen it cuts off the bottom tool bar as well as parts of the upper tool bar (the toolbar which contains the 'Disconnect, Actions, Settings, etc' buttons).  I suspect that may be caused by the overall increase in size I mentioned above?

At the MeshCentral logon screen (no matter if you use the normal UI or the new bootstrap version) the Enter key no longer works for when you are entering the logon password.  I tried in both Edge and Chrome thinking maybe it was some recent browser change.  Same issue with both browsers.  I have to click the Logon button to complete the logon procedure.  Whereas, previously, I could just hit Enter after entering the password. 
 
I'll keep updating as I find other quirks, oddities, etc.

**Reply by si458** [reply]
> At the MeshCentral logon screen (no matter if you use the normal UI or the new bootstrap version) the Enter key no longer works for when you are entering the logon password. I tried in both Edge and Chrome thinking maybe it was some recent browser change. Same issue with both browsers. I have to click the Logon button to complete the logon procedure. Whereas, previously, I could just hit Enter after entering the password.

this issue will be because of this https://github.com/Ylianst/MeshCentral/pull/6434 #6428
i did explain it might breaks things and it appears so, so plz open an issue for this as its effecting a normal setup

**Reply by PathfinderNetworks** [reply]
Will do.  Opening the issue now.

**Comment by ronald-mendoza** [comment]
Reporting an issue; with the new interface, the MeshCentral server update functionality is broken. **My Server** > **General** > **Check server version** shows the versions available, but none of the checkboxes can be selected to continue with an update.

**Reply by si458** [reply]
can you send a screenshot of what you are seeing please?
we did release 1.1.34 but it has bugs in so we had to roll back to 1.1.33, but we have just released 1.1.35 about an hour or 2 ago

**Reply by ronald-mendoza** [reply]
![image](https://github.com/user-attachments/assets/b291b340-372c-4853-b3bb-e435609b9720)

This was occurring for me on 1.1.33. I noticed it when I tried to update to 1.1.35, and the behavior persists on 1.1.35.

Edit: Meant to mention seeing this on Firefox 133.0 and Chrome 131.0.6778.86.

**Reply by si458** [reply]
yeh i see what you mean! it does actually make then enabled, but then disables them for no reason, will fix it!

**Reply by si458** [reply]
fixed https://github.com/Ylianst/MeshCentral/commit/d9262f7c9da03881fd25c363f717f452d06e117d

**Comment by jiriteach** [comment]
Great work on this. I see the additional bootstrap themes are included - is there any ability to easily change the bootstrap theme? Thanks

**Reply by si458** [reply]
already in the works, just not ready yet https://github.com/Ylianst/MeshCentral/issues/6202

**Comment by jiriteach** [comment]
Quick question re-upgrade
On 1.1.33 - 

<img width="437" alt="Screenshot 2024-12-02 at 1 08 33 PM" src="https://github.com/user-attachments/assets/3d22fbf6-70b4-4495-9550-57622d4a2ac4">

I see above that its been fixed. Assuming that I'll have to manually upgrade. 
Whats the best approach to manually upgrade?

Running on Ubuntu so assume a download and replace /opt/meshcentral? 

Sorry for the question - just new to MeshCentral.

Thanks

**Reply by si458** [reply]
Try restart restarting your meshcentral server, then checking the update panel as it should show 1.1.35 now!
Remember tho if ur using docker don't use the web panel! Update the docker way by changing the tag!

**Reply by jiriteach** [reply]
Perfect - thanks. Worked.
